packages/amplify-category-custom/src/utils/generate-cfn-from-cdk.ts

```diff
@@ -0,0 +1,14 @@
+import * as path from 'path';
+import * as cdk from '@aws-cdk/core';
+import { JSONUtilities, pathManager } from 'amplify-cli-core';
+import { categoryName } from './constants';
+
+export async function generateCloudFormationFromCDK(resourceName: string) {
+  const targetDir = path.join(pathManager.getBackendDirPath(), categoryName, resourceName);
+  const { cdkStack } = require(path.resolve(path.join(targetDir, 'build', 'cdk-stack.js')));
+
+  const customStack: cdk.Stack = new cdkStack(undefined, undefined, undefined, { category: categoryName, resourceName });
+
+  // @ts-ignore
```

Comment:
why @ts-ignore ?

Reply:
Since _toCloudFormation() is an internal/pvt function.